backend/src/api/database-migration.ts

```diff
@@ -558,6 +558,21 @@ class DatabaseMigration {
       await this.$executeQuery('ALTER TABLE `statistics` ADD min_fee FLOAT UNSIGNED DEFAULT NULL');
       await this.updateToSchemaVersion(66);
     }
+
+    if (databaseSchemaVersion < 67 && config.MEMPOOL.NETWORK === "liquid") {
+      // Drop and re-create the elements_pegs table
+      await this.$executeQuery('DROP table IF EXISTS elements_pegs;');
```

Comment:
Instead of drop and recreate... `TRUNCATE TABLE elements_pegs` should be enough

Reply:
Yes truncate would be better, but in this case I also added a primary key on `(txid, txindex)` to `elements_pegs` so drop / create also adds the primary key to the table

Reply:
In that case it's better to do a TRUNCATE and then
`ALTER TABLE elements_pegs ADD PRIMARY KEY (txid, txindex);`
To make the migration flow consistent.